[2023-12-13T07:44:38.613+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v1.create_postgres_table scheduled__2023-04-25T00:00:00+00:00 [queued]>
[2023-12-13T07:44:38.624+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v1.create_postgres_table scheduled__2023-04-25T00:00:00+00:00 [queued]>
[2023-12-13T07:44:38.625+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-12-13T07:44:38.661+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): create_postgres_table> on 2023-04-25 00:00:00+00:00
[2023-12-13T07:44:38.667+0000] {standard_task_runner.py:57} INFO - Started process 801 to run task
[2023-12-13T07:44:38.670+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v1', 'create_postgres_table', 'scheduled__2023-04-25T00:00:00+00:00', '--job-id', '292', '--raw', '--subdir', 'DAGS_FOLDER/dag_w_postgres_operator.py', '--cfg-path', '/tmp/tmp9t745t7d']
[2023-12-13T07:44:38.672+0000] {standard_task_runner.py:85} INFO - Job 292: Subtask create_postgres_table
[2023-12-13T07:44:38.730+0000] {task_command.py:416} INFO - Running <TaskInstance: dag_with_postgres_operator_v1.create_postgres_table scheduled__2023-04-25T00:00:00+00:00 [running]> on host 6bd2d0daa4c2
[2023-12-13T07:44:38.854+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='vuhta' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v1' AIRFLOW_CTX_TASK_ID='create_postgres_table' AIRFLOW_CTX_EXECUTION_DATE='2023-04-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-04-25T00:00:00+00:00'
[2023-12-13T07:44:38.855+0000] {sql.py:274} INFO - Executing: 
            create table if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
            )
        
[2023-12-13T07:44:38.869+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-12-13T07:44:38.882+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-12-13T07:44:38.890+0000] {sql.py:418} INFO - Running statement: 
            create table if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
            )
        , parameters: None
[2023-12-13T07:44:38.900+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v1, task_id=create_postgres_table, execution_date=20230425T000000, start_date=20231213T074438, end_date=20231213T074438
[2023-12-13T07:44:38.963+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-13T07:44:38.988+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
